CMakeLists.txt

```diff
@@ -77,6 +77,9 @@ set(QGIS_APP_NAME "qgis" CACHE STRING "The main app name and bundle name")
 math(EXPR QGIS_VERSION_INT "${CPACK_PACKAGE_VERSION_MAJOR}*10000+${CPACK_PACKAGE_VERSION_MINOR}*100+${CPACK_PACKAGE_VERSION_PATCH}")
 message(STATUS "QGIS version: ${COMPLETE_VERSION} ${RELEASE_NAME} (${QGIS_VERSION_INT})")
 
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
This suggests we should be able to remove this line with the `target_compile_features`

https://stackoverflow.com/a/70667652